src/DocBlock/Annotation.php

```diff
@@ -185,6 +166,33 @@ public function getTag()
         return $this->tag;
     }
 
+    /**
+     * @return TypeExpression
+     *
+     * @internal until 2.17
```

Comment:
I don't think we need to do this. Just write internal, and we can delete this tag in 2.17 if we want, or indeed not!

Reply:
I prefer keeping the version so that if we forget to remove the tag, one still knows this is not internal anymore on 2.17 and higher.